src/Pages/BulkPackageSale/BulkPackageWrap/BulkPackageList.js

```diff
@@ -1,15 +1,29 @@
 import React, { Component } from "react";
-import "./BulkPackageWrap.scss";
+import "./BulkPackageList.scss";
 
 class BulkPackageList extends Component {
   render() {
-    const { product } = this.props;
+    const { product, handleActive, index, id } = this.props;
+
     return (
       <div className="BulkPackageList">
-        <li className="option-item">
-          <div className="item-text-wrapper">
+        <li
+          className={
+            product.active === true ? "option-item-active" : "option-item"
+          }
+        >
```

Comment:
```suggestion
        <li
          className={
            product.active ? "option-item active" : "option-item"
          }
        >
```
1. product.active 자체가 true / false 값을 가지기 때문에 굳이 === true라고 안써주셔도 됩니다
2. active를 별개로 분리해주시면 좋겠네요. option-item에 대한 공통 속성들을 주고 active 클래스 명에 바뀔 속성 지정해주시면 더 좋을거 같습니다

Reply:
![image](https://user-images.githubusercontent.com/65284962/88771407-2b5f2780-d1ba-11ea-8cc2-f417f1da847e.png)
수정했습니다.